src/pipeline_v3/legacy_backup/llama_index_files/test_llamaindex_postgresql.py

```
@@ -0,0 +1,415 @@
+                index_manager.vector_store = mock_vector_store
+
+                # Test adding document
+                doc_id = str(uuid.uuid4())
+                content = "Test document content for PostgreSQL backend"
+                metadata = {"source": "test.pdf", "category": "datasheet"}
+
+                # Mock embedding model
+                index_manager.embedding_model = Mock()
+
+                # Add document
+                success = index_manager.add_document(
```

Ruff (0.11.9)

Local variable `success` is assigned to but never used

Remove assignment to unused variable `success`

(F841)